virt-v2v/cold/entrypoint.go

```diff
@@ -127,6 +128,25 @@ func buildCommand() []string {
 				virtV2vArgs = append(virtV2vArgs, "--mac", macToIp)
 			}
 		}
+		// Adds LUKS keys, if exist.
+		if _, err := os.Stat(LUKSDIR); os.IsNotExist(err) {
+			// do nothing
+		} else {
+			if err != nil && !os.IsNotExist(err) {
+				fmt.Println("Error accessing the LUKS directory ", err)
+				os.Exit(1)
+			}
+			files, err := getFilesInPath(LUKSDIR)
+			if err != nil {
+				fmt.Println("Error reading files in LUKS directory ", err)
+				os.Exit(1)
+			}
+			for _, file := range files {
+				virtV2vArgs = append(virtV2vArgs, "--key", fmt.Sprintf("all:file:%s", file))
+			}
+		}
+		// fallback to clevis, harmless
+		virtV2vArgs = append(virtV2vArgs, "--key", "all:clevis")
```

Comment:
i put it anyway, but we can say we set it only when a secret exists (even without relevant passphrase). i think this way it is now is good enough - being ignored from virt-v2v if no LUKS needed.

Reply:
are you sure there is no performance implication for specifying this? it doesn't seem right to specify this without checking whether clevis is defined, unless we're sure about the implications

Reply:
i think if there is no encryption, nothing will happen. if there is, my guess is that it will first try provided keys as they precedent args, if non of them fit, it will try clevis. sure it can harm performance. what i wrote is just theory how it works, we can ask richard or remove this part completely

Reply:
I'd just drop it - it's counter intuitive to specify that when the user doesn't provide any indication for having encrypted disks